images/macos/toolsets/toolset-14.json

```diff
@@ -3,24 +3,105 @@
+                },
+                {
+                    "link": "15.2",
+                    "version": "15.2.0+15C500b",
+                    "sha256": "04E93680C6DDBEC84666531BE412DE778AFC8EAC6AB2037F4C2BE7290818B59B",
+                    "install_runtimes": ["all"]
+                },
+                {
+                    "link": "15.1",
+                    "version": "15.1.0+15C65",
+                    "sha256": "857D8DB537BAC82BF99DE0E1D3895D214D4D02101C1340CEF3DAF6E821BA1D05",
+                    "install_runtimes": ["all"]
+                },
+                {
+                    "link": "15.0.1",
+                    "version": "15.0.1+15A507",
+                    "sha256": "5AC17AE6060CAFC3C7112C6DA0B153450BE21F1DE6632777FBA9FBC9D999C9E8",
+                    "symlinks": ["15.0"],
+                    "install_runtimes": ["all"]
+                }
             ]
         },
         "arm64":{
             "versions": [
-                { "link": "16.2", "version": "16.2+16C5032a", "install_runtimes": ["iOS -buildVersion 18.2", "watchOS", "tvOS"], "sha256": "0e367d06eb7c334ea143bada5e4422f56688aabff571bedf0d2ad9434b7290de"},
-                { "link": "16.1", "version": "16.1+16B40", "install_runtimes": ["iOS", "watchOS", "tvOS"], "sha256": "8ca961d55981f983d21b99a95a6b0ac04905b837f6e11346ee86d28f12afe720"},
-                { "link": "15.4", "version": "15.4.0+15F31d", "install_runtimes": "true", "sha256": "82d3d61804ff3f4c7c82085e91dc701037ddaa770e542848b2477e22f4e8aa7a"},
-                { "link": "15.3", "version": "15.3.0+15E204a", "install_runtimes": "true", "sha256": "f13f6a2e2df432c3008e394640b8549a18c285acd7fd148d6c4bac8c3a5af234"},
-                { "link": "15.2", "version": "15.2.0+15C500b", "install_runtimes": "true", "sha256": "04E93680C6DDBEC84666531BE412DE778AFC8EAC6AB2037F4C2BE7290818B59B"},
-                { "link": "15.1", "version": "15.1.0+15C65", "install_runtimes": "true", "sha256": "857D8DB537BAC82BF99DE0E1D3895D214D4D02101C1340CEF3DAF6E821BA1D05"},
-                { "link": "15.0.1", "version": "15.0.1+15A507", "symlinks": ["15.0"], "install_runtimes": "true", "sha256": "5AC17AE6060CAFC3C7112C6DA0B153450BE21F1DE6632777FBA9FBC9D999C9E8"}
+                {
+                    "link": "16.2",
+                    "version": "16.2+16C5032a",
+                    "sha256": "0e367d06eb7c334ea143bada5e4422f56688aabff571bedf0d2ad9434b7290de",
+                    "install_runtimes": [
+                        { "iOS": "18.2" },
+                        { "watchOS": "default" },
+                        { "tvOS": "default" },
+                        { "visionOS": "2.2" }
+                    ]
+                },
+                {
+                    "link": "16.1",
+                    "version": "16.1+16B40",
+                    "sha256": "8ca961d55981f983d21b99a95a6b0ac04905b837f6e11346ee86d28f12afe720",
+                    "install_runtimes": ["all"]
```

Comment:
Thinking more about it maybe we could use work `default` to install everything and install nothing if `install_runtimes` property is omitted? I do not have a strong opinion here but I am trying to reduce amount of "key words" that someone would have to keep in mind...
- "default" feels like a good word and it used to be a default behavior
- and omitting  `install_runtimes` could be a good signal to not install anything

Reply:
> "default" feels like a good word and it used to be a default behavior

We have documentation that describes the behaviour. We know that "by default" does not always behave predictably, for example. Whatever word we choose, there will be misinterpretations until it is explained in an accessible way.

> and omitting install_runtimes could be a good signal to not install anything

If we leave the ability to remove the field, then sooner or later someone will forget about adding it and we will only discover it on release. It is better to let the script slap our hands for every possible error at the build stage.